README.md

```diff
@@ -8,7 +8,13 @@ I'm creating this project as part of my studies with [Manchester Codes](https://
 
 ## Roadmap
 
-I should hopefully have this all finished by the end of week 4 of this module, which I think is Christmas Day 🙃
+~~I should hopefully have this all finished by the end of week 4 of this module, which I think is Christmas Day 🙃~~
```

Comment:
This is good start to a readme, it is missing a few important things though. Some things that would make it much better:
- installation instructions (ie cloning the repo and what commands are available),
- what technologies are required to use it, what the application dependencies are (`JavaScript` and `Jest` in this case)
- a description of what the app actually does and what its features are. What are the rules for how the pet is fed, exercised, how long it will live, what are the max and min values for things like hunger, fitness etc.
- how the app is tested,

As well as your name it would be good to link your socials (esp linkedIn) to the repo.